runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -563,7 +563,14 @@
 	TWISTED_LEAGUE_RELIC_2(10050),
 	TWISTED_LEAGUE_RELIC_3(10051),
 	TWISTED_LEAGUE_RELIC_4(10052),
-	TWISTED_LEAGUE_RELIC_5(10053);
+	TWISTED_LEAGUE_RELIC_5(10053),
+
+	/**
+	 * Status of the PvP Spec Orb
+	 * 0 = Disabled
+	 * 1 = Enabled
```

Comment:
This javadoc does not match the implementation of `inPvp()`. I would expect it to read more like the following:
```suggestion
	 * Whether the Special Attack orb is disabled due to being in a PvP area
	 *
	 * 0 = Enabled (player is not in PvP)
	 * 1 = Disabled (player in in PvP)
	 *
	 * @see https://oldschool.runescape.wiki/w/Minimap#Special_attack_orb
```